listenbrainz/db/dump_manager.py

```diff
@@ -157,12 +168,24 @@ def create_incremental(location, threads, dump_id):
             write_hashes(dump_path)
         except IOError as e:
             current_app.logger.error('Unable to create hash files! Error: %s', str(e), exc_info=True)
-            return
+            sys.exit(-1)
+
+        try:
+            if not sanity_check_dumps(dump_path, 6):
+                return sys.exit(-1)
+        except IOError as e:
+            sys.exit(-1)
 
         # if in production, send an email to interested people for observability
         send_dump_creation_notification(dump_name, 'incremental')
 
+        # Write the DUMP_ID file so that the FTP sync scripts can be more robust
+        print(os.path.join(dump_path, "DUMP_ID.txt"))
+        with open(os.path.join(dump_path, "DUMP_ID.txt"), "w") as f:
+            f.write("%s %s incremental\n" % (end_time.strftime('%Y%m%d-%H%M%S'), dump_id))
```

Comment:
Why a newline here and not in full? https://github.com/metabrainz/listenbrainz-server/pull/1178/files#diff-e4f7644db199bd1cedfa7296a724b02679b34ea30b8920fdbae2eff15153cd9cR129

Reply:
Good catch. fixing.

Reply:
Fixed